pkg/config/shuttleplan.go

```diff
@@ -67,23 +67,27 @@ func (p *ShuttlePlanConfiguration) Load(planPath string) *ShuttlePlanConfigurati
 }
 
 // FetchPlan so it exists locally and return path to that plan
-func FetchPlan(plan string, projectPath string, localShuttleDirectoryPath string, uii ui.UI, skipGitPlanPulling bool) string {
+func FetchPlan(plan string, projectPath string, localShuttleDirectoryPath string, uii ui.UI, skipGitPlanPulling bool, planArgument string) string {
+	if isPlanArgumentAPlan(planArgument) {
+		uii.Infoln("Using overloaded plan %v", planArgument)
+		return FetchPlan(getPlanFromPlanArgument(planArgument), projectPath, localShuttleDirectoryPath, uii, skipGitPlanPulling, "")
+	}
+	
 	switch {
 	case plan == "":
 		uii.Verboseln("Using no plan")
 		return ""
 	case git.IsGitPlan(plan):
 		uii.Verboseln("Using git plan at '%s'", plan)
-		return git.GetGitPlan(plan, localShuttleDirectoryPath, uii, skipGitPlanPulling)
+		return git.GetGitPlan(plan, localShuttleDirectoryPath, uii, skipGitPlanPulling, planArgument)
 	case isMatching("^http://|^https://", plan):
-		panic("plan not valid: http is not supported yet")
+		panic(fmt.Sprintf("Plan '%v' is not valid: non-git http/https is not supported yet", plan))
```

Comment:
If we explicitly haven't implemented HTTP(S) support, I think we should omit it in the documentation as well? This is confusing to me.

Reply:
We have it, just only for git repositories.